libdevcore/JSON.h

```diff
@@ -23,22 +23,97 @@
 #pragma once
 
 #include <json/json.h>
+#include <sstream>
+#include <memory>
 
 namespace dev
 {
 
+namespace json
+{
+
+/// StreamWriterBuilder that can be constructed with specific settings
+class StreamWriterBuilder : public Json::StreamWriterBuilder {
+public:
+	explicit StreamWriterBuilder(const std::map<std::string, std::string> &_settings)
```

Comment:
Please put the `const` after the type

Reply:
As far as I know `const T&` is just an alternative notation of `T& const`. I know that this notation is sometimes preferred, but why exactly?